### ISSUE TITLE
toJS on a class with a Date observable

### ISSUE THREAD

**Issue by Brendan-csel** [mobxjs/mobx#377]
If there is a Date observable on a class instance then toJS turns it into some kind of Object - but not a date.  JSfiddle is here: https://jsfiddle.net/af4faun5/1/

```
const {observable, toJS} = mobx;

class Todo {
    @observable title;
    @observable dueDate = new Date();
    constructor(title) {
        this.title = title;
    }
}

var todo = new Todo("Something to do");

console.log(todo.dueDate);  // original is a date...
var obj = toJS(todo)
console.log(obj.dueDate);  // ..but not after toJS

```

**Comment by mweststrate** [comment]
Thanks for reporting! That is a bug indeed.

**Comment by mweststrate** [comment]
Fixed in `2.3.4`. Thanks for reporting again
